### ISSUE TITLE
sails no-frontend creates EJS warning

### ISSUE THREAD

**Issue by mikermcneil** [balderdashy/sails-generate-new#21]
_From @crh3675 on December 29, 2015 21:53_

I was running a Sails application using `--no-frontend`. Upon startup, this message occurs:

```
Could not find module: ejs in path: /opt/wx-api/node_modules
```
I updated the `config/views.js` as follows to supress:

```
engine: { ext : 'null', fn : function(){} },
```
Everything works fine just questioning if there is another way.

Also, when running Mocha using `sails.lift()` from within tests, the application complains that it can't find Grunt.  I installed as a devDependency but would like to remove if I can.

<pre><code>
var Sails = require('sails');

process.chdir(APPLICATION_PATH);

before(function(done){

   this.timeout(5000);

   Sails.lift({
       port : 12345,
       log: {
           level: 'error'
       }

   }, function(err, app) {

       sails = app;
       
       if (err) return done(err);

       done(err, sails);
   });
});
</code></pre>

_Copied from original issue: balderdashy/sails#3455_

**Comment by mikermcneil** [comment]
_From @crh3675 on December 29, 2015 22:2_

Update: I just added

```
hooks : { grunt : false }
````

to Sails.lift(options) and was able to remove

**Comment by mikermcneil** [comment]
@crh3675 re: grunt-- spot on.  You can also stick that in your .sailsrc file or configure it via env variables (e.g. `sails_hooks__grunt=false sails lift`).  

re: views, what version of NPM and Sails are you using?  We were relying on the nested node_modules structure prior to NPM 3 (i.e. to symlink the `ejs` dep in sails so you don't have to `npm install` when running `sails new`).  The latest 0.12 rc just sticks copies the dependency down, but I'm curious if there's a bug there that's only showing itself when the no-frontend option is enabled.

Thanks!

**Comment by sailsbot** [comment]
Thanks for posting, @mikermcneil.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
